grunt/svg_inline.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+
+var inline = require( "web-resource-inliner" );
+var a = require( "async" );
+
+module.exports = function ( grunt )
+{
+    grunt.registerTask( "svg_inline", "Inline SVG use", function ()
+    {
+        var done = this.async();
+        var src = [
+            "modules/*/templates/*.html",
+            "!modules/_app/templates/index.html"
```

Comment:
Probably. Something to consider is that ngtemplates also ignores the index file, so that would need to change as well. Either that or copy references the `.inlined` index file now.